lib/taiko.js

```diff
@@ -470,29 +491,13 @@ module.exports.setLocation = async(options) => {
  * @param {string} url - URL to navigate page to.
  * @param {boolean} [options.waitForNavigation=false] - Skip to navigation - default is true
  * @param {Object} options - {timeout:5000, headers:{'Authorization':'Basic cG9zdG1hbjpwYXNzd29y2A=='}} Default timeout is 30 seconds to override set options = {timeout:10000}, headers to override defaults.
+ * @param {string} [options.waitForEvents = ['loadEventFired','networkIdle','firstMeaningfulPaint']] Defaults to firstMeaningfulPaint
```

Comment:
@NivedhaSenthil Can you also include networkAlmostIdle

Reply:
Updated docs with available events to wait for 👍